src/middlewares/one-of.spec.ts

```diff
@@ -159,3 +159,41 @@ describe('error message', () => {
     });
   });
 });
+
+describe('imperatively run oneOf', () => {
+  it('context should have errors', async () => {
+    const req: InternalRequest = {
+      body: { foo: true },
+    };
+
+    const middleware = oneOf([check('foo').isInt()]);
+    await middleware.run(req);
+
+    const context = getOneOfContext(req);
+    expect(context.errors.length).toEqual(1);
+  });
+
+  it('should throw an error if ContextRunner throws', async () => {
+    const req: InternalRequest = {
+      body: { foo: true },
+    };
+    const error = new Error();
+    const spy = jest.spyOn(ContextRunnerImpl.prototype, 'run').mockRejectedValue(error);
+
+    const middleware = oneOf([check('foo').isBoolean()]);
+    await expect(middleware.run(req)).rejects.toThrow(error);
+    spy.mockRestore();
+  });
+
+  it('successful validation', async () => {
```

Comment:
Can you just make this read more phrasal?
E.g. `sets no error in context when successful` or similar to how it's in other tests, like 
https://github.com/express-validator/express-validator/blob/7f15d2d963574f34e0e312a96228b7b0693df1cf/src/middlewares/one-of.spec.ts#L73